### ISSUE TITLE
_render_sprites.so: undefined symbol: cudaGetDevice

### ISSUE THREAD

**Issue by lqiang2003cn** [e2crawfo/auto_yolo#8]
i successfully executed the install.sh, but when run:

python yolo_air_run.py local

i got this issue:

tensorflow.python.framework.errors_impl.NotFoundError: /root/PycharmProjects/auto_yolo_2/auto_yolo/tf_ops/render_sprites/_render_sprites.so: undefined symbol: cudaGetDevice

any idea? thanks a lot!

**Comment by SayanChaki** [comment]
Yeah I have been facing the same issue, for some reason the file _render_sprites.so doesn't seem to generate
